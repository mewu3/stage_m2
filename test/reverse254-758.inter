> LC506451.1 |Coxsackievirus A6 CV-A6/HVN15.144_HAI_PHONGVNM/2015 genomic RNA, complete genome|254-758
atgcggccaatcctaactgcggagcacatgccctcaatcca-gggg-gtg
> MT364381.1 |Human rhinovirus A12 strain JX-RVA12-41-2017, complete genome|254-758
atgtggctaatcctaaccccgctgctattgcacacgaacca-gtgt-gtg
> MT338567.1 |Enterovirus C117 strain JX-C117-40-2017, complete genome|254-758
atgtggctaatcctaaccaccgaacccgtggttgcaatcca-gcaa-cta
> MT791934.1 |Enterovirus D68 isolate CF125132-FRA-2016, complete genome|254-758
atgcggctaatctcaaccatggagcaagtgctcacaaacca-gtga-gtt
> MN914198.1 |Enterovirus C strain 04-1378, complete genome|254-758
--------------------------------------------------
> MN914199.1 |Enterovirus C strain 03-4166, complete genome|254-758
--------------------------------------------------
> MN914200.1 |Enterovirus C strain 03-4107, complete genome|254-758
--------------------------------------------------
> MN914204.1 |Enterovirus C strain 04-4444, complete genome|254-758
--------------------------------------------------
> MN914206.1 |Enterovirus C strain 03-0100, complete genome|254-758
--------------------------------------------------
> MN845762.1 |Coxsackievirus A6 isolate 3/HLJ/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MN845763.1 |Coxsackievirus A6 isolate 148/NX/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MN845765.1 |Coxsackievirus A6 isolate 57/JL/CHN/2012, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MK106189.1 |Coxsackievirus A6 strain 54203/HeB/CHN/2012, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MK106196.1 |Coxsackievirus A6 strain FH233/SX/CHN/2013, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MK106208.1 |Coxsackievirus A6 strain 075/GS/CHN/2011, complete genome|254-758
atgcggataatcctaactgcagagcgcatgccctcaatcca-gggg-gtg
> MK106209.1 |Coxsackievirus A6 strain 233/HeB/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MK106210.1 |Coxsackievirus A6 strain 046/SX/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcataccctcaaccca-gggg-gtg
> MK106211.1 |Coxsackievirus A6 strain 058/SX/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MN749144.1 |Human rhinovirus A45 strain 12I7, complete genome|254-758
atgtggctaatcctaaccccgcagctgtagtgtgcaatcca-gcac-att
> MN749147.1 |Human rhinovirus 1B strain 12J7, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MN749150.1 |Human rhinovirus 1B strain 12L1, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MN749153.1 |Human rhinovirus 1B strain 12M3, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MN749156.1 |Human rhinovirus 1B strain 12O2, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MN749157.1 |Human rhinovirus 1B strain 12O3, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MN749158.1 |Human rhinovirus A66 strain 12O6, complete genome|254-758
atgtggctaaccctaaccctgcagctagtgcatgaaatcca-tcat-gtt
> MN481403.1 |Enterovirus C isolate BE-18-7469-L, complete genome|254-758
atgtggataatcctaaccaccgatcctatgttcgcaatcca-gcgt-tcg
> MN228693.1 |Human rhinovirus C3 strain USA/2019/CA-RGDS-1060, complete genome|254-758
atgtggctaaccctaaccccgtagctgttgcatgtaatcca-acat-gta
> MN369030.1 |Rhinovirus C strain SC9810, complete genome|254-758
atgcggctaatc-caaccccacatctgttgcacacaatcca-gtgt-gta
> MN369033.1 |Rhinovirus C strain SC9941, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaatcca-gcat-gtg
> MN369035.1 |Human rhinovirus A54 strain SC9916, complete genome|254-758
-------------------------------------------------g
> MN369038.1 |Human rhinovirus C11 strain SC9849, complete genome|254-758
atgcggctaatcttaaacctgcagccatcgcatgcatgcca-gcgt-gta
> MN369040.1 |Human rhinovirus A12 strain SC9843, complete genome|254-758
atgtggctaatcctaaccccgctgctattgcacacgaacca-gtgt-gtg
> MN369043.1 |Human rhinovirus B86 strain SC9785, complete genome|254-758
atgcggctaaccctaaccccggagccttgtggcacaagcca-gtga-cat
> MN369044.1 |Human rhinovirus B35 strain SC9780, complete genome|254-758
atgcggctaacctcaaccccggagccttgtattgcaatcca-gcaa-tat
> MN369045.1 |Human rhinovirus A53 strain SC9746, complete genome|254-758
atgtggctaaccttaaacccgcagccaaggtgtgcaagcca-gcat-att
> MN306020.1 |Rhinovirus A strain SC0218, complete genome|254-758
atgcggctaaccttaaacccgcagccattactcacaatcca-gtga-gcg
> MN306023.1 |Rhinovirus C strain SC0323, complete genome|254-758
atgtggctaatcctaaccccgtagctgttgcatgcaaccca-gcat-gta
> MN306026.1 |Rhinovirus C strain SC0364, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaatcca-gcat-gtg
> MN306035.1 |Rhinovirus C strain SC0645, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaatcca-gtat-gtt
> MN306049.1 |Human rhinovirus A53 strain SC0894, complete genome|254-758
atgtggctaaccttaaacccgcagccaaggtgtgcaagcca-gcat-att
> MF662679.1 |Enterovirus A71 strain R176/YN/CHN/2009, complete genome|254-758
atgcggcaaatcctaactgcggagcacacgcccacaaacca-gcgg-gta
> MN181514.1 |Enterovirus B strain 2014_AFP_NGR, complete genome|254-758
-----------------------------------aagcca-gcgg-gta
> MH330336.1 |Rhinovirus C strain NIV1722289, complete genome|254-758
atgcggctaatc-caaccctgcagccattacatggaagcca-ccat-gtg
> MF285621.1 |Coxsackievirus A6 isolate CVA6/S546/BJ/CHN/2010, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MF285622.1 |Coxsackievirus A6 isolate CVA6/S845/BJ/CHN/2010, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaaccca-gggg-gtg
> MF285623.1 |Coxsackievirus A6 isolate CVA6/S942/BJ/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MF285624.1 |Coxsackievirus A6 isolate CVA6/S945/BJ/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MF285625.1 |Coxsackievirus A6 isolate CVA6/S983/BJ/CHN/2011, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> MH229997.1 |Human enterovirus C105 isolate R2759, complete genome|254-758
atgtggctaatcctaaccactgatcccacgcacgcaaacca-gtgt-gta
> MK089787.1 |UNVERIFIED: Human enterovirus C117 isolate CQ6747, complete genome|254-758
atgtggctaatcccaaccaccgaacccgtggttgcaatcca-gcaa-cta
> MH118045.1 |Coxsackievirus A10 strain 13-2185-1, complete genome|254-758
atgcggctaatcttaactgcggagcacatgccttcaaccca-ggag-gtg
> MH118060.1 |Coxsackievirus A10 strain 11-691-1, complete genome|254-758
atgcggataatcctaactgcggagcacatactcacaaacca-gtga-gct
> MH118084.1 |Coxsackievirus A10 strain 09-3318-1, complete genome|254-758
atgcggataatcctaactgcggagcacatactcacaaacca-gtga-gct
> MH144604.1 |Echovirus E24 strain 13-2053-1_E24, complete genome|254-758
--------------------------------------------------
> MH716374.1 |Enterovirus A71 isolate B5-117, complete genome|254-758
atgcggccaatcctaactgcggagcacatgccttcaaccca-gagg-gta
> MK520815.1 |Human rhinovirus C32 strain USA/CA/RGDS-2016-1008, complete genome|254-758
atgtggctaatcctaaccccgtatctatcgcatgcgcacca-gcat-gta
> MK279354.1 |Rhinovirus C41 strain USA/CA/2016-RGDS-1006, complete genome|254-758
atgcggctaatc-taaccccgcagctgtggcacacaagcca-gtgt-gta
> MK105982.1 |Enterovirus D68 isolate 2123800027, complete genome|254-758
atgcggctaaccttaaacctgcagccattgctcacaatcca-gtga-gtt
> MH752985.1 |Human rhinovirus C42 strain USA/CA/2016-23027, complete genome|254-758
atgcggctaatc-caaccctgcagccattacatggaagcca-ccat-gtg
> MH718269.1 |Enterovirus A71 strain USA/CO/2018-23026, complete genome|254-758
--------------------------------------------------
> MH128992.1 |Enterovirus C109 strain USA/FL/2016-21002, complete genome|254-758
atgtggataatcctaaccactgaacctacgggcgcaaacca-gcgt-ctg
> MH128993.1 |Enterovirus C109 strain USA/FL/2016-21003, complete genome|254-758
atgtggataatcctaaccactgaacctacgggcgcaagcca-gcgt-ctg
> MG881840.1 |Human rhinovirus C15 strain HRV/C/BuenosAires/Arg/001/2017, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaagcca-gcat-gtg
> MG950178.1 |Human rhinovirus C56 strain USA/2016/CA-RGDS-1002, complete genome|254-758
atgcggctaatc-caaccccacagctatcgcatggattcca-ccat-gta
> MG950179.1 |Human rhinovirus C56 strain USA/2017/CA-RGDS-1003, complete genome|254-758
atgcggctaatc-caaccccacagctatcgcatggattcca-ccat-gta
> MF678295.1 |Coxsackievirus B2 isolate NSW-V06-2008-CVB2, complete genome|254-758
-----gctaatcccaactgcggagcagatacccacacacca-gtgg-gca
> MF678298.1 |Echovirus E25 isolate NSW-V09-2008-ECHO25, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatgcca-gtgg-gca
> MF678303.1 |Coxsackievirus A9 isolate NSW-V14-2009-CVA9, complete genome|254-758
-----gctaatcctaactgcggagcaggcaccctcaagcca-gagg-gca
> MF678304.1 |Coxsackievirus B3 isolate NSW-V15-2012-CVB3, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatacca-gtgg-gca
> MF678305.1 |Coxsackievirus B2 isolate NSW-V16-2005-CVB2, complete genome|254-758
-----gctaatcccaactgcggagcatgcgacctcaaacca-gagg-atg
> MF678306.1 |Coxsackievirus B2 isolate NSW-V17-2006-CVB2, complete genome|254-758
-----gctaatcccaactgcggagcagatacccacacacca-gtgg-gca
> MF678307.1 |Echovirus E6 isolate NSW-V18-2007-ECHO6, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatacca-gtgg-gca
> MF678308.1 |Coxsackievirus B5 isolate NSW-V19-2008-CVB5, complete genome|254-758
-----gctaatcctaactgcggagcacatgccctcaaccca-gggg-gta
> MF678314.1 |Coxsackievirus B3 isolate NSW-V25-2015-CVB3, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatgcca-gtgg-gca
> MF678317.1 |Coxsackievirus B2 isolate NSW-V29-2008-CVB2, complete genome|254-758
-----gctaatcccaactgcggagcagacacccacacacca-gtgg-gca
> MF678319.1 |Coxsackievirus B4 isolate NSW-V31-2010-CVB4, complete genome|254-758
-----gctaatcctaactgtggagcatgcgcccccactcca-gagg-gta
> MF678323.1 |Echovirus E7 isolate NSW-V35A-2006-ECHO7, complete genome|254-758
-----gctaatcccaactgcggagcagacacccacaagcca-gtgg-gca
> MF678324.1 |Coxsackievirus B3 isolate NSW-V35B-2006-CVB3, complete genome|254-758
--------------------------------------------------
> MF678325.1 |Echovirus E7 isolate NSW-V36-2012-ECHO7, complete genome|254-758
-----gctaatcccaactgcggagcagacacccacaagcca-gtgg-gca
> MF678326.1 |Coxsackievirus B3 isolate NSW-V37-2013-CVB3, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatacca-gtgg-gca
> MF678327.1 |Coxsackievirus B3 isolate NSW-V38A-2009-CVB3, complete genome|254-758
--------------------------------------------------
> MF678331.1 |Coxsackievirus B2 isolate NSW-V42-2008-CVB2, complete genome|254-758
-----gctaatcccaactgcggagcagatacccacacacca-gtgg-gca
> MF678334.1 |Coxsackievirus A2 isolate NSW-V45-2005-CVA2, complete genome|254-758
-----gctaatcctaactgcggagcacacgccctcaatcca-gggg-gtg
> MF678335.1 |Echovirus E30 isolate NSW-V46-2008-ECHO30, complete genome|254-758
-----gctaatcctaactgcggagcagattcccacacgcca-gtgg-gca
> MF678336.1 |Echovirus E9 isolate NSW-V47A-2005-ECHO9, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatacca-gtgg-gca
> MF678337.1 |Echovirus E30 isolate NSW-V47B-2005-ECHO30, complete genome|254-758
-----gctaatcctaactgcggagcagatacccacatacca-gtgg-gca
> MF678339.1 |Echovirus E7 isolate NSW-V49-2012-ECHO7, complete genome|254-758
-----gctaatcccaactgcggagcagacacccacaagcca-gtgg-gca
> MF678341.1 |Coxsackievirus B5 isolate NSW-V51-2008-CVB5, complete genome|254-758
-----gctaatcctaactgcggagcacatgccctcaaccca-gggg-gta
> MF678346.1 |Coxsackievirus A9 isolate NSW-V56-2006-CVA9, complete genome|254-758
-----gctaatcctaactgcggagcatgcgcccccgttcca-gagg-gta
> MF678347.1 |Coxsackievirus B4 isolate NSW-V57-2007-CVB4, complete genome|254-758
-----gctaatcctaactgtggagcatgcgcccccattcca-gagg-gta
> MF678348.1 |Echovirus E25 isolate NSW-V58-2010-ECHO25, complete genome|254-758
-----gctaatcccaactgcggggcagatacccacatgcca-gtgg-gca
> MF973193.1 |Human rhinovirus A1 strain 5Q1, complete genome|254-758
atgcggctaaccttaaacctgcagccatggctcataagcca-atga-gtt
> MF973194.1 |Human rhinovirus A1 strain 7A2, complete genome|254-758
atgcggctaaccttaaacctgcagccatagctcataagcca-atga-gtt
> MF806525.1 |Human rhinovirus C47 isolate CA-RGDS-1001, complete genome|254-758
atgcggctaatc-caaccccacagctattgcacacaaacca-gtgt-gta
> MF775366.1 |Human rhinovirus C53 strain USA/CO/2014-20993, complete genome|254-758
atgcggctaaccctaaccccgtagctgtggtgtgcacgcca-gcac-aag
> KX494871.1 |Human rhinovirus B91 isolate ZJ2015_001, complete genome|254-758
atgcggctaacctaaaccctggagccttgtgacacaatcca-gtgt-tgg
> KY629935.1 |Human rhinovirus 59 strain 16-J2, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacataagcca-atgt-gta
> KY460514.1 |Human rhinovirus A101 isolate TW-00006-2010, complete genome|254-758
--------------------------------------------------
> KY684757.1 |Human rhinovirus B3 strain SC2220, complete genome|254-758
atgcggctaaccctaaccctggagccttgagtcacaatcca-gtga-tgg
> KY342346.1 |Human rhinovirus A22 strain HRV-A22/Lancaster/2015, complete genome|254-758
atgcggctaaccttaaacctgcagccattgctcacaatcca-gtga-gtt
> KY369875.1 |Human rhinovirus A98 strain SC176, complete genome|254-758
atgtggctaaccttaaacccgtagctagtgcacacaagcca-gtgt-gta
> KY369878.1 |Human rhinovirus C43 strain SC174, complete genome|254-758
atgtggctaatcctaaccccgcagctactacacataagcca-atgt-gca
> KY369880.1 |Rhinovirus C strain SC261, complete genome|254-758
atgcggctaatctt-accccgcagctgttgcacacaatcca-gtgt-gta
> KY369881.1 |Human rhinovirus C2 strain SC9735, complete genome|254-758
atgcggctaatc-caaccccacagctattgcacacaaacca-gtgt-gta
> KY369882.1 |Rhinovirus C strain SC9795, complete genome|254-758
atgcggctaatctt-accccgcagctgttgcacacaatcca-gtgt-gta
> KY369883.1 |Human rhinovirus B97 strain SC1079, complete genome|254-758
atgcggctaaccctaaccccggagcccagcaacataatcca-atgt-tgt
> KY369885.1 |Human rhinovirus A22 strain SC1029, complete genome|254-758
atgcggctaaccttaaacctgcagccattgcttacaatcca-gtga-gtt
> KY369889.1 |Human rhinovirus A101 strain SC1124, complete genome|254-758
atgtggctaatcctaaccccgtatccattgcacacaaacca-gtgt-gtc
> KY369891.1 |Human rhinovirus A101 strain SC996, complete genome|254-758
atgtggctaatcctaaccccgtatccattgcacacaagcca-gtgt-gtt
> KY369892.1 |Human rhinovirus A101 strain SC1004, complete genome|254-758
atgtggctaatcctaaccccgtatccattgcacacaaacca-gtgt-gtc
> KY369893.1 |Human rhinovirus A59 strain SC1095, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacataagcca-atgt-gta
> KY369894.1 |Human rhinovirus A82 strain SC9702, complete genome|254-758
atgcggctaaccttaaacccgcagccattgctcacaatcca-gtgg-gta
> KY369895.1 |Human rhinovirus A59 strain SC1038, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacataagcca-atgt-gta
> KY369897.1 |Human rhinovirus A101 strain SC1104, complete genome|254-758
atgtggctaatcctaaccccgtatccattgcacacaaacca-gtgt-gtc
> KY369898.1 |Human rhinovirus 1B strain SC9813, complete genome|254-758
atgcggctaaccttaaacccgcagccatggttcacaaacca-gtga-act
> KX891411.1 |Human rhinovirus A16 isolate KC939 clone pR16.939, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacacaaacca-gtgt-gta
> KT779410.1 |Coxsackievirus A6 isolate Changchun046/CHN/2013, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtt
> KX398052.1 |Human rhinovirus A12 isolate p211, complete genome|254-758
atgtggctaatcctaaccccgctgctattgcacacgaacca-gtgt-gtg
> KP890662.1 |Rhinovirus C isolate 7383-MY-10, complete genome|254-758
atgtggctaatcctaaccccgtagctgttgcatgtaaccca-acat-gtc
> KP890663.1 |Rhinovirus C isolate 3805-MY-10, complete genome|254-758
atgcggctaatc-caaccctgcagccattgcatggaaacca-ccat-gta
> KT726984.1 |Human rhinovirus A39 strain 2-D5-TW, complete genome|254-758
atgtggctaaccttaaacctgcagccaatgcacacaaccca-gtgt-gta
> KJ675505.1 |Rhinovirus C isolate 1570-MY-10, complete genome|254-758
atgcggctaatc-caaccctgcagccattacatggaagcca-ccat-gtg
> KJ675506.1 |Rhinovirus C isolate 8713-MY-10, complete genome|254-758
atgcggctaatc-caaccccgcagctgtagtatgcaagcca-gcat-atg
> KJ675507.1 |Rhinovirus C isolate 3430-MY-10, complete genome|254-758
atgtggctaatcctaaccctgcagctgttgcacacaagcca-gtgt-gta
> KP289438.1 |Coxsackievirus B5 isolate CV-B5/P727/2013/China, complete genome|254-758
atgcggctaatcttacatgcgcagcgggcacccacaaacca-gtggaaac
> KR815824.1 |Human enterovirus C104 isolate kvv585-16-TS, complete genome|254-758
atgtggataatcctaaccactgatcctatgttcgcaatcca-gcgt-tca
> KP233830.1 |Coxsackievirus B5 isolate 3936/09, complete genome|254-758
atccggctaatcctaactgcggagcacataccctcaaccca-gggg-gca
> KF734978.1 |Rhinovirus C isolate 1515-MY-10, complete genome|254-758
atgtggctaatcctaaccccgtagctgttgcatgcaaccca-gcat-gta
> KM279379.1 |Coxsackievirus A6 strain CA6/CC13/57, complete genome|254-758
atgcggataatcccaactgcggagcgcatgccctcaatcca-gggg-gtt
> KJ186974.1 |Enterovirus A71 strain C4a-01585-TW-2011, complete genome|254-758
--------------------------------------------------
> KF015730.1 |Echovirus E30 VP1 protein gene, partial cds|254-758
--------------------------------------------------
> KF015731.1 |Echovirus E12 VP1 protein gene, partial cds|254-758
--------------------------------------------------
> KF015732.1 |Echovirus E20 VP1 protein gene, partial cds|254-758
--------------------------------------------------
> KF015733.1 |Human rhinovirus A22 VP1 protein gene, partial cds|254-758
--------------------------------------------------
> KF958308.1 |Rhinovirus B isolate 3039, complete genome|254-758
atgcggctaacctaaaccctggagccttggagcacaagcca-gtgc-ttg
> KF958309.1 |Rhinovirus B isolate R93, complete genome|254-758
atgcggctaaccctaaccccggagccttgcatcacaatcca-gtga-tgt
> KF958310.1 |Rhinovirus C isolate 6331, complete genome|254-758
atgcggctaatc-caaccccacagctattgcacacaagcca-gtgt-gta
> KF958311.1 |Rhinovirus C isolate 2536, complete genome|254-758
atgcggctaatc-caaccccgcagctgtggcacacaatcca-gtgt-gta
> KF193624.1 |Coxsackievirus A16 isolate HN09-02, complete genome|254-758
atacggctaatcctaactgcggagcgcgcaccctcaaccca-gggg-gcg
> KC880366.1 |Human poliovirus 1 isolate TJK35363 clone 4, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880367.1 |Human poliovirus 1 isolate TJK35363 clone 6, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880368.1 |Human poliovirus 1 isolate TJK35363 clone 19, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880369.1 |Human poliovirus 1 isolate TJK35363 clone 2, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880370.1 |Human poliovirus 1 isolate RUS39270, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880371.1 |Human poliovirus 1 isolate KAZ38374, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880372.1 |Human poliovirus 1 isolate TJK35325, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880373.1 |Human poliovirus 1 isolate RUS36716, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880374.1 |Human poliovirus 1 isolate TKM37184, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880375.1 |Human poliovirus 1 isolate TJK35389 clone 1, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880376.1 |Human poliovirus 1 isolate TJK35389 clone 9, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880377.1 |Human poliovirus 1 isolate TJK36567, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880378.1 |Human poliovirus 1 isolate RUS37255, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880379.1 |Human poliovirus 1 isolate TJK35453, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> KC880380.1 |Human poliovirus 1 isolate RUS36815, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-ggag-gtg
> KC880381.1 |Human poliovirus 1 isolate TJK37339, complete genome|254-758
atgcggccaatcctaaccacggagcaagcgccttcaaccca-gggg-gtg
> JX678288.1 |Human enterovirus C118 isolate CQ5185, complete genome|254-758
atgtggctaatcctaaccactgatcctgtgagcacacgcca-gtgc-cca
> JX982253.1 |Human enterovirus C104 strain Pavia259-7712, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982254.1 |Human enterovirus C104 strain Pavia260-9210, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982255.1 |Human enterovirus C104 strain Pavia263-11230, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982256.1 |Human enterovirus C104 strain Pavia264-16291, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982257.1 |Human enterovirus C104 strain Pavia261-9570, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982258.1 |Human enterovirus C104 strain Pavia68-10804B, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX982259.1 |Human enterovirus C104 strain Pavia262-11228, complete genome|254-758
atgtggataatcccaaccactgatcctacgttcgcaatcca-gcgt-tca
> JX961708.1 |Human enterovirus C118 strain ISR10, complete genome|254-758
atgtggctaatcctaaccactgatcctgtgagcacacacca-gtgc-cta
> JX961709.1 |Human enterovirus C118 strain ISR38, complete genome|254-758
atgtggctaatcctaaccactgatcctgtgagcacgcacca-gtgc-cta
> JX262382.1 |Human enterovirus C117 strain LIT22, complete genome|254-758
atgtggctaatcccaaccaccgaacccgtggttgcaatcca-gcaa-cta
> JQ950555.1 |Human enterovirus 71 strain 26M/AUS/4/99/GuaR1, complete genome|254-758
--------------------------------------------------
> JQ964234.1 |Human coxsackievirus A6 strain HN421, complete genome|254-758
atgcggataatcctaactgcggagcgcatgccctcaatcca-gggg-gtg
> JN205461.1 |Human rhinovirus C strain WA823M02, complete genome|254-758
atgtggctaatcctaaccccgtatctatcgcatgcacacca-gcat-gta
> JQ354992.1 |Human coxsackievirus A16 strain Ningbo.CHN/028-2/2009, complete genome|254-758
atgcggctaatcttaactgcggagcgcgtaccctcaatcca-gggg-gca
> JN695050.1 |Human coxsackievirus B5 strain CVB5/CC10/16, complete genome|254-758
--------------------------------------------------
> JN695051.1 |Human coxsackievirus B5 strain CVB5/CC10/17, complete genome|254-758
--------------------------------------------------
> JF260924.1 |Human coxsackievirus A17 isolate CV-A17_67610, complete genome|254-758
atgcggccaatcccaaccgcggagcaggtggttgcaaccca-gcaa-ccg
> JF317013.1 |Human rhinovirus C isolate LZ269, complete genome|254-758
atgcggctaatc-caaccccacatctgtcgcacacaatcca-gtgt-gta
> JF317014.1 |Human rhinovirus C isolate LZY79, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaatcca-gcat-gtg
> JF317016.1 |Human rhinovirus C isolate LZ651, complete genome|254-758
atgtggctaatcctaaccccgtagctgttgcatgcaaccca-gcat-gta
> JF317017.1 |Human rhinovirus C isolate LZY101, complete genome|254-758
atgcggctaatc-caaccctgcagccattgcatgggaacca-ccat-gta
> HQ647168.1 |Human enterovirus 71 isolate EV92_07, complete genome|254-758
agacaatcaattgggaaggggtaactagccaacacatcatc------ccc
> HQ647170.1 |Human enterovirus 71 isolate EV124_07, complete genome|254-758
agacaatcaattgggaaggggtaactagccaacacatcatc------ccc
> HQ423143.1 |Human enterovirus 71 strain KM186/09, complete genome|254-758
atgcggttaatcctaactgcggagcacacccccacaaacca-gcgg-gta
> HQ891925.1 |Human enterovirus 71 strain Shanghai 36-2009, complete genome|254-758
atgcggccaatcccaactgcggagcacacgcccacaagcca-gcgg-gta
> GU219984.1 |Human rhinovirus C strain HRV-C15 isolate W10, complete genome|254-758
atgcggctaatc-caaccccacagctatggcatgcaagcca-gcat-gtg
> GQ865517.1 |Human enterovirus 109 isolate NICA08-4327, complete genome|254-758
atgtggataatcctaaccactgaacctacgggcgcaaacca-gcgt-ctg
> GU396280.1 |Human enterovirus 71 isolate EV71/Lanzhou01, complete genome|254-758
atgcggctgatcctaactgcggagcacacgcccacaagcca-gcgg-gta
> GQ415051.1 |Human rhinovirus A isolate hrv-A101, complete genome|254-758
atgtggctaatcctaaccccgtatccattgcacacaaacca-gtgt-gtc
> GQ415052.1 |Human rhinovirus A isolate hrv-A101-v1, complete genome|254-758
atgtggctaatcctaaccccgtatccattacacacaagcca-gtgt-gtt
> FJ842158.1 |Human poliovirus 3 strain FIN84-60212, complete genome|254-758
atgcggctaatctcaaccacggagcaggtggtcacaaacca-gtga-ttt
> FJ842159.1 |Human poliovirus 3 strain FIN84-2493, complete genome|254-758
atgcggctaatctcaaccacggagcaggtagtcacaaacca-gtga-ctt
> GQ323774.1 |Human rhinovirus C strain QCE, complete genome|254-758
atgtggctaatcctaaccccgtagctgttgcatgtaatcca-acat-gta
> EU840952.2 |Human rhinovirus C11 strain CL-170085, complete genome|254-758
atgcggctaatctt-accccacagctgttgcacgcaaacca-gcgt-gta
> GQ223227.1 |Human rhinovirus C isolate N4, complete genome|254-758
atgcggctaatc-ttaccccacagctattgcatggaagcca-ccat-gtg
> FJ445117.1 |Human rhinovirus 13 isolate F03, complete genome|254-758
atgcggctaaccctaaccctgcagccattgcccacaatcca-gtgg-gtt
> FJ445137.1 |Human rhinovirus 52 isolate F10, complete genome|254-758
atgcggctaacctaaaccctggagccttggagcacaagcca-gtgc-tta
> FJ445139.1 |Human rhinovirus 54 isolate F05, complete genome|254-758
atgtggctaaccttaaacccgtagctagtgcacacaatcca-gtgt-gtg
> FJ445141.1 |Human rhinovirus 57 isolate fs ship#1-hrv-57, complete genome|254-758
atgtggctaaccctaaccctgcagctagtgcatacaagcca-gtat-gtt
> FJ445158.1 |Human rhinovirus 81 isolate F06, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacacaaacca-gtgt-gta
> FJ445159.1 |Human rhinovirus 81 isolate F07, complete genome|254-758
atgtggctaaccttaaacctgcagccagtgcacacaaacca-gtgt-gta
