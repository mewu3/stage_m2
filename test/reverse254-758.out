ATGCGGCTAATCC	2856
TAACCATGGAGCA	167
CTAACCTTAACCC	45
